x-pack/plugins/fleet/scripts/install_all_packages/install_all_packages.ts

```diff
@@ -57,7 +57,7 @@ async function deletePackage(name: string, version: string) {
 
 async function getAllPackages() {
   const res = await fetch(
-    `${REGISTRY_URL}/search?experimental=true&kibana.version=${KIBANA_VERSION}`,
+    `${REGISTRY_URL}/search?prerelease=true&kibana.version=${KIBANA_VERSION}`,
```

Comment:
should we try to install the prerelease version of packages here or only the GA ones?

Reply:
I think we may want to make this configurable but that can probably be done later.